hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/commit/staging/StagingCommitter.java

```diff
@@ -556,14 +574,6 @@ protected void abortJobInternal(CommitContext commitContext,
    * @throws IOException IO failure
    */
   protected void deleteDestinationPaths(JobContext context) throws IOException {
-    Path attemptPath = getJobAttemptPath(context);
-    ignoreIOExceptions(LOG,
-        "Deleting Job attempt Path", attemptPath.toString(),
-        () -> deleteWithWarning(
-            getJobAttemptFileSystem(context),
-            attemptPath,
-            true));
```

Comment:
Removed these as `attemptPath` here is a sub dir under `${UUID}/staging-uploads`, which has already been deleted in  `deleteStagingUploadsParentDirectory()`.